File: official/docs/java/webhooks/RetrieveWebhook.java
import com.easypost.EasyPost;

public class RetrieveWebhook {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        Webhook webhook = Webhook.retrieve("hook_...");

        System.out.println(webhook);
    }
}

File: official/docs/csharp/current/webhooks/retrieve-webhook.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

            var client = EasyPost.Client(apiKey);

            Webhook webhook = await client.Webhook.Retrieve("hook_...");

            Console.WriteLine(JsonConvert.SerializeObject(webhook, Formatting.Indented));
        }
    }
}
